### PR TITLE
[TR] Fix Behats on master

### PR DESCRIPTION
**Pull request by jjanvier**
CI is running

### CODE REVIEW

File: features/Context/Page/Product/Edit.php [hunk 1]
```diff
@@ -1715,17 +1715,4 @@ public function getCategoryTree()
 
         return $this->getElement('Category tree');
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function openPanel($panel)
```
Comment (jjanvier) [inline features/Context/Page/Product/Edit.php]: removed this method that doesn't work for published completeness and made the original more generic to handle localisation (ie: `I open the History panel` works even in French)

**Comment by skeleton** [comment]
GTM

File: features/Context/Page/Product/Edit.php (hunk 1 above)
Comment (juliensnz) [inline features/Context/Page/Product/Edit.php]: still don't understand why we created it in first place ^^ But nice catch :)

**Comment by solivier** [comment]
GTM

**Comment by willy-ahva** [comment]
GTM

**Pull request merged**